presto-docs/src/main/sphinx/sql/create-function.rst

```diff
@@ -0,0 +1,84 @@
+===============
+CREATE FUNCTION
+===============
+
+Synopsis
+--------
+
+.. code-block:: none
+
+    CREATE [ OR REPLACE ] FUNCTION
+    qualified_function_name (
+      parameter_name parameter_type
+      [, ...]
+    )
+    RETURNS return_type
+    [ COMMENT function_description ]
+    [ LANGUAGE language ]
+    [ DETERMINISTIC | NOT DETERMINISTIC ]
+    [ RETURNS NULL ON NULL INPUT | CALLED ON NULL INPUT ]
+    [ RETURN expression ]
+
+
+Description
+-----------
+
+Create a new function with the specified definition.
+
+Each function is uniquely identified by its qualified function name
+and its parameter type list. ``qualified_function_name`` needs to be in
+the format of ``catalog.schema.function_name``.
+
+To create a function, the function namespace as identified by
+``catalog.schema`` needs to managed by a
+:doc:`/admin/function-namespace-managers`.
+
+The optional ``OR REPLACE`` clause causes the query to quietly replace
+the existing function if a function with the identical signature (function
+name with parameter type list) exists.
+
+The ``return_type`` needs to match the actual type of the routine body
+``expression``, without performing type coercion.
+
+A set of routine characteristics can be specified to decorate the
+function and specify its behavior. Each kind of routine
+characteristic can be specified at most once.
+
+============================ ======================== ================================================================
+Routine Characteristic       Default Value            Description
```

Comment:
For all of these, what other (non-default) options are supported

Reply:
The other values are available in the `Synopsis` section. Placing them in the table may jeopardize readability.